src/Aggregation/DateHistogram.php

```diff
@@ -7,10 +7,67 @@
  *
  * @see https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-datehistogram-aggregation.html
  */
-class DateHistogram extends Histogram
+class DateHistogram extends AbstractSimpleAggregation
```

Comment:
That would be a BC break.

Reply:
I thought BC breaks were allowed

Reply:
Sometimes but should not ;)
I think in this case, we can avoid it or I'm missing something?

Reply:
so, what should we do?

Reply:
Wouldn't this be helpful (https://github.com/ruflin/Elastica/pull/1874#discussion_r528717001) to avoid changing the inheritance?

Reply:
Ok I think I got the point.
DateHistogram `interval` parameter is deprecated but still useful for `Histogram`.
Can you confirm it's what you're addressing by changing the inheritance chain?

Reply:
yes. We could move methods if we want to keep those implementation. 
Maintaining BC on interfaces and userland API yes, but should we maintain BC on abstract internal classes?